tahoe_idp/magiclink_helpers.py

```diff
@@ -0,0 +1,45 @@
+from datetime import timedelta
+from uuid import uuid4
+
+from django.conf import settings as djsettings
+from django.http import HttpRequest
+from django.utils import timezone
+from django.utils.crypto import get_random_string
+from tahoe_idp import magiclink_settings as settings
+from tahoe_idp.magiclink_utils import get_client_ip, get_url_path
+from tahoe_idp.models import MagicLink, MagicLinkError
+
+
+def create_magiclink(
+    username: str,
+    request: HttpRequest,
+    redirect_url: str = '',
+) -> MagicLink:
+    limit = timezone.now() - timedelta(seconds=settings.LOGIN_REQUEST_TIME_LIMIT)  # NOQA: E501
+    over_limit = MagicLink.objects.filter(username=username, created__gte=limit)
+    if over_limit:
+        raise MagicLinkError('Too many magic login requests')
```

Comment:
The error says `too many` but it's comparing with `created`. Something seems wrong.

Reply:
it is correct. but the name `created` is not accurate. I've renamed it to `created_on`